helmfile/charts/networkpolicy-common/templates/prometheus/allow-allow-cert-manager-resolver.yaml

```diff
@@ -0,0 +1,30 @@
+{{ if .Values.monitoring.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-cert-manager-resolver
+  namespace: monitoring
+spec:
+  podSelector:
+    matchLabels:
+      acme.cert-manager.io/http01-solver: "true"
+  ingress:
+    - from:
+      {{- if .Values.global.ingressUsingHostNetwork }}
+        {{- range $ip := .Values.global.nodes.ips }}
+        - ipBlock:
+            cidr: {{ $ip }}
+        {{- end }}
+      {{- else }}
+        - namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: ingress-nginx
+          podSelector:
+            matchLabels:
+              app.kubernetes.io/component: controller
+              app.kubernetes.io/instance: ingress-nginx
+      {{- end }}
+      ports:
+      - port: 8089
+        protocol: TCP
+{{- end }}
```

Comment:
I think this will be covered in a general policy done in this PR https://github.com/elastisys/compliantkubernetes-apps/pull/1199